data/battle_anim_scripts.s

```diff
@@ -1492,22 +1492,22 @@ Move_HEAL_BLOCK:
 	loadspritegfx ANIM_TAG_BLUE_STAR
 	monbg ANIM_TARGET
 	createsoundtask SoundTask_PlaySeChangingVolume, SE_M_ABSORB_2, SOUND_PAN_TARGET, 256, -16, 0, 2
-	createsprite gHealingBlueStarSpriteTemplate, ANIM_TARGET, 2, 0, -5, 1, 0
+	createspriteontargets_onpos gHealingBlueStarSpriteTemplate, ANIM_TARGET, 2, 2, 0, -5, ANIM_TARGET, 0
```

Comment:
Heal Block animation is a bit broken in singles, it plays as if there were two targets.

Reply:
> Heal Block animation is a bit broken in singles, it plays as if there were two targets.

Added IsBattlerAlive checks for each target, consolidated checks to new function, `GetBattleAnimMoveTargets`